packages/vaadin-grid/src/vaadin-grid.js

```diff
@@ -919,6 +921,8 @@ class GridElement extends ElementMixin(
         cell._instance.setProperties(model);
       }
     });
+
+    this._updateDetailsCellHeight(row);
```

Comment:
Would it work if we instead invoked this at the end of `_toggleDetailsCell()`?

Reply:
To make it work we need to run renderers in `_toggleDetailsCell` before updating height. That in turn will lead to the second render [here](https://github.com/vaadin/web-components/pull/2076/files#diff-f55be1687931f6b0777e240bb62e036552dd9f76d85bc316689f8b1258d7a79bR912-R922).

Moving the rendering logic back into `_toggleDetailsCell` will also require getting the model in the method.

Reply:
Ok got it. Maybe it's better to keep it this way then.